README.md

```diff
@@ -9,21 +9,35 @@
 
 [简体中文](README_zh_CN.md)
 
-OpenPAI is an open source platform that provides complete AI model training and resource management capabilities, it is easy to extend and supports on-premise, cloud and hybrid environments in various scale.
+**OpenPAI [v1.0.0](./RELEASE_NOTE.md#2020-v1.0.0) has been released!**
+
+With the release of v1.0, OpenPAI is switching to a more robust, more powerful and lightweight architecture. OpenPAI is also becoming more and more modular so that the platform can be easily customized and expanded to suit new needs. OpenPAI also provides many AI user-friendly features, making it easier for end users and administrators to complete daily AI tasks. 
```

Comment:
Change v1.0 to `v1.0.0` to remain consistency?

Reply:
v1.0 is more generic, we might have to have v1.0.1 with quick fixes, with this v1.0, we don't need to change it by then...